core/src/main/java/org/mule/runtime/core/internal/message/DefaultEventBuilder.java

```diff
@@ -405,9 +404,8 @@ public FlowConstruct getFlowConstruct() {
     }
 
     /**
-     * Invoked after deserialization. This is called when the marker interface
-     * {@link DeserializationPostInitialisable} is used. This will get invoked after the object
-     * has been deserialized passing in the current MuleContext when using either
+     * Invoked after deserialization. This is called when the marker interface {@link DeserializationPostInitialisable} is used.
```

Comment:
Shoudl open an issue to kill DeserializationPostInitialisable or at least move it to comp.